site/.gitignore

```diff
@@ -0,0 +1,7 @@
+## Python venv
+/pyvenv.cfg
```

Comment:
is the '/' prefix needed? `.gitignore` under `python` directory doesn't seem to use it.

Reply:
Not strictly required, but this way it's just explicit (otherwise it would match any `pyvenv.cfg`, not just the one for "our" venv).